.github/workflows/ci.yml

```diff
@@ -232,7 +232,7 @@ jobs:
             !:trino-cassandra,
             !:trino-hive,!:trino-orc,!:trino-parquet,
             !:trino-mongodb,!:trino-kafka,!:trino-elasticsearch,
-            !:trino-redis,
+            !:trino-redis,!:trino-pulsar,
```

Comment:
yeah, previously there's only unit test so we thought it can be merged with other test, now that we added query test and it takes 10+ mins, probably better to run as a separate step. updated

Reply:
Thanks, please make exclusion for trino-pulsar also in separate line.